static/app/views/settings/organizationRateLimits/organizationRateLimits.tsx

```diff
@@ -47,84 +51,107 @@ function OrganizationRateLimit({organization}: OrganizationRateLimitProps) {
   };
 
   return (
-    <div>
-      <SettingsPageHeader title={t('Rate Limits')} />
+    <Feature
+      organization={organization}
+      features={['legacy-rate-limits']}
+      renderDisabled={_ => (
+        <EmptyMessage
+          icon={<IconInfo size="xl" />}
+          title={t(`Legacy rate limits are not available in this organization.`)}
+        >
+          {tct(
+            `See our [link:Billing Quota Management] docs for more information about managing your organization's quota.`,
+            {
+              link: (
+                <Link
+                  to="https://docs.sentry.io/pricing/quotas/"
```

Comment:
links can only be used to link to pages within our app
```suggestion
                <ExternalLink
                  href="https://docs.sentry.io/pricing/quotas/"
```

Reply:
THANKS fixing